### PR TITLE
Feature/interleaved 2q rb validation

### PR DESCRIPTION
**Pull request by deanpoulos**
- Fixed likely bug in interleaved RB
- Validated the the sequences are correct
- Added fidelity calculation convenience function for interleaved gate fidelity
- Not yet tested on a device.

### CODE REVIEW

File: Quantum-Control-Applications/Superconducting/Two-Flux-Tunable-Transmons/Use Case 2 - Two-Qubit-Randomized-Benchmarking/two_qubit_rb/RBResult.py [hunk 1]
```diff
@@ -32,6 +32,53 @@ def plot_hist(self, n_cols=3):
             self.data.state.sel(circuit_depth=circuit_depth).plot.hist(ax=ax, xticks=range(4))
         plt.tight_layout()
 
-    def plot_fidelity(self):
+    def plot_decay(self):
```
Comment (arthurostrauss) [inline Quantum-Control-Applications/Superconducting/Two-Flux-Tunable-Transmons/Use Case 2 - Two-Qubit-Randomized-Benchmarking/two_qubit_rb/RBResult.py:35]: I would add a small docstring describing what it returns if this is a public method accessible to the user

File: Quantum-Control-Applications/Superconducting/Two-Flux-Tunable-Transmons/Use Case 2 - Two-Qubit-Randomized-Benchmarking/two_qubit_rb/RBResult.py [hunk 2]
```diff
@@ -32,6 +32,53 @@ def plot_hist(self, n_cols=3):
...
         fidelity = (self.data.state == 0).sum(("repeat", "average")) / (self.num_repeats * self.num_averages)
         fidelity.rename("fidelity").plot.line()
+
+    def plot_fidelity(self):
```
Comment (arthurostrauss) [inline Quantum-Control-Applications/Superconducting/Two-Flux-Tunable-Transmons/Use Case 2 - Two-Qubit-Randomized-Benchmarking/two_qubit_rb/RBResult.py:39]: Same comment here, just to clarify how plot_decay differentiates from plot_fidelity, would be nice to have a small docstring.

**Review by arthurostrauss — CHANGES_REQUESTED** [review]
Looks good! I would just add a few docstrings on the public methods of the class that the user might want to use directly, so that one knows exactly what is being computed/plotted.

File: Quantum-Control-Applications/Superconducting/Two-Flux-Tunable-Transmons/Use Case 2 - Two-Qubit-Randomized-Benchmarking/two_qubit_rb/RBResult.py (hunk 1 above)
Comment (deanpoulos) [reply Quantum-Control-Applications/Superconducting/Two-Flux-Tunable-Transmons/Use Case 2 - Two-Qubit-Randomized-Benchmarking/two_qubit_rb/RBResult.py]: Hey Arthur, thanks for the suggestion.

I added docstrings everywhere and also renamed the methods to make them more clear. Please tell me what you think now.

File: Quantum-Control-Applications/Superconducting/Two-Flux-Tunable-Transmons/Use Case 2 - Two-Qubit-Randomized-Benchmarking/two_qubit_rb/RBResult.py (hunk 2 above)
Comment (deanpoulos) [reply Quantum-Control-Applications/Superconducting/Two-Flux-Tunable-Transmons/Use Case 2 - Two-Qubit-Randomized-Benchmarking/two_qubit_rb/RBResult.py]: Done.  Please tell me what you think

**Review by arthurostrauss — APPROVED** [review]

**Pull request merged**